src/edu-benefits/schema.js

```diff
@@ -0,0 +1,436 @@
+// Relocate options-for-select.
+import options from '../common/options-for-select';
```

Comment:
Now that we're in a separate repo, should we change the name of options-for-select? It doesn't make much sense in here. That file has static data that could be used in a number of contexts. We could probably change it in vets-website, too.